Doc/library/fcntl.rst

```diff
@@ -176,17 +175,17 @@ The module defines the following functions:
    method are accepted as well) of the file to lock or unlock, and *cmd*
    is one of the following values:
 
-   * :const:`LOCK_UN` -- unlock
-   * :const:`LOCK_SH` -- acquire a shared lock
-   * :const:`LOCK_EX` -- acquire an exclusive lock
+   * :const:`!LOCK_UN` -- unlock
+   * :const:`!LOCK_SH` -- acquire a shared lock
+   * :const:`!LOCK_EX` -- acquire an exclusive lock
 
-   When *cmd* is :const:`LOCK_SH` or :const:`LOCK_EX`, it can also be
-   bitwise ORed with :const:`LOCK_NB` to avoid blocking on lock acquisition.
-   If :const:`LOCK_NB` is used and the lock cannot be acquired, an
+   When *cmd* is :const:`!LOCK_SH` or :const:`!LOCK_EX`, it can also be
+   bitwise ORed with :const:`!LOCK_NB` to avoid blocking on lock acquisition.
+   If :const:`!LOCK_NB` is used and the lock cannot be acquired, an
    :exc:`OSError` will be raised and the exception will have an *errno*
-   attribute set to :const:`EACCES` or :const:`EAGAIN` (depending on the
+   attribute set to :data:`~errno.EACCES` or :data:`~errno.EAGAIN` (depending on the
    operating system; for portability, check for both values).  On at least some
-   systems, :const:`LOCK_EX` can only be used if the file descriptor refers to a
+   systems, :const:`!LOCK_EX` can only be used if the file descriptor refers to a
```

Comment:
Eh, providing a hyperlink to two or three lines above seems both cluttered and unnecessary. I think it should stay as-is. It would be different if `LOCK_EX` was defined elsewhere (like the `errno` constants).

(I was chided about something similar in earlier revisions of other library docs.)

Reply:
Yeah, I might have been the one chiding you, haha. And indeed, with your change to locate the constant definition right above within this function, no need for this then :+1: